wiki/Medals/de.md

```diff
@@ -1,74 +1,59 @@
----
-outdated: true
----
-
 # Medaillen
 
-**Medaillen**, auch früher bekannt als **Achievements** (dt. Erfolge oder Errungenschaften), sind kleine Embleme auf der Profilseite, welche anzeigen, dass jemand eine triviale Herausforderung gemeistert hat. Zur Zeit sind sie in 5 Hauptgruppen unterteilt: **Beatmap Packs**, **Skill**, **Dedication**, **Hush-Hush** und **Mod Introduction**.
+**Medaillen**, auch früher bekannt als *Achievements* (zu Dt. Erfolge oder Errungenschaften), sind Embleme auf der Profilseite, welche anzeigen, dass der Benutzer ein Ziel erreicht hat. Zur Zeit können **261** Medaillen freigeschaltet werden in 8 unterschiedlichen Kategorien: [Beatmap Packs](#beatmap-packs), [Skill](#skill), [Dedication](#dedication), [Mod Introduction](#mod-introduction), [Hush-Hush](#hush-hush), [Beatmap Spotlights](#beatmap-spotlights), [Seasonal Spotlights](#seasonal-spotlights) und [Beatmap Challenge Packs](#beatmap-challenge-packs).
 
 ## Beatmap Packs
 
-**Beatmap Packs** Erfolge können erlangt werden, indem aus jedem Mapset eines [Theme Beatmap Packs](https://osu.ppy.sh/p/packlist?t=t) eine beliebige Schwierigkeitsstufe gespielt wird. Die Beatmaps müssen im [osu!](/wiki/Game_mode/osu!) Modus von osu! gespielt werden.
+**Beatmap Packs Medaillen** sind Medaillen, die beim Abschließen bestimmter [Thema-Beatmap-Pakete](https://osu.ppy.sh/beatmaps/packs?type=theme) oder [Beatmap-Pakete von Künstlern/Alben](https://osu.ppy.sh/beatmaps/packs?type=artist) überreicht werden. Diese Medaillen können erlangt werden, indem jede Beatmap in irgendeiner Schwierigkeitsstufe innerhalb des entsprechenden Beatmap-Pakets mindestens einmalig gemeistert wird.
 
-Zur Zeit gibt es 16 dieser Erfolge, welche sich nochmal in 4 Untergruppen aufteilen lassen:
+Alle Beatmaps müssen im [osu!](/wiki/Game_mode/osu!)-Spielmodus bezwungen werden, um sich für das Abzeichen zu qualifizieren, sofern nicht anders angegeben. Die Verwendung von entweder [Mods zur Verringerung der Schwierigkeit](/wiki/Game_modifier#difficulty-reduction) oder [Mods zur Erhöhung der Schwierigkeit](/wiki/Game_modifier#difficulty-increase) ist erlaubt.
 
-- Video Game Pack
-- Anime Pack
-- Internet! Pack
-- Rhythm Game Pack
+Es ist jedoch erwähnenswert, dass der Inhalt aller Video Game Packs, Anime Packs, Internet! Packs und Rhythm Game Packs sich seit [ihrer ersten Bekanntmachung im Jahr 2008](https://osu.ppy.sh/community/forums/topics/1853) dramatisch verändert haben. Siehe [originale Beatmap-Pakete](/wiki/Medals/Legacy_beatmap_packs) für die Liste dieser Beatmap-Pakete in ihrem alten/originalen Zustand.
```

Comment:
```suggestion
Es ist jedoch erwähnenswert, dass der Inhalt aller Video Game Packs, Anime Packs, Internet! Packs und Rhythm Game Packs sich seit [ihrer ersten Bekanntmachung im Jahr 2008](https://osu.ppy.sh/community/forums/topics/1853) dramatisch verändert hat. Siehe [originale Beatmap-Pakete](/wiki/Medals/Legacy_beatmap_packs) für die Liste dieser Beatmap-Pakete in ihrem alten/originalen Zustand.
```

Reply:
```suggestion
Es ist jedoch erwähnenswert, dass sich der Inhalt aller Video Game Packs, Anime Packs, Internet! Packs und Rhythm Game Packs seit [ihrer ersten Bekanntmachung im Jahr 2008](https://osu.ppy.sh/community/forums/topics/1853) dramatisch verändert hat. Siehe [originale Beatmap-Pakete](/wiki/Medals/Legacy_beatmap_packs) für die Liste dieser Beatmap-Pakete in ihrem alten/originalen Zustand.
```
Sounds better to me when reading it out loud.

Reply:
Yeah, that's better.